<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(239, 236, 227);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(26, 16, 84);</string>
   </property>
   <property name="text">
    <string>Generate Monthly Revenue Report</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="inputsFrame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#inputsFrame {
        border: 2px solid #1A1054;
        border-radius: 12px;
   
    }</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QComboBox" name="monthField">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QComboBox {
        border: 2px solid #1A1054;
        border-radius: 5px;
        padding: 3px;
background-color: rgb(255, 255, 255);
	color: rgb(26, 16, 84);
    }</string>
    </property>
    <property name="placeholderText">
     <string>Month</string>
    </property>
    <item>
     <property name="text">
      <string>January</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>February</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>March</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>April</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>May</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>June</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>July</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>August</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>September</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>October</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>November</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>December</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="yearField">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QSpinBox {
        border: 2px solid #1A1054;
        border-radius: 5px;
        padding: 3px;
background-color: rgb(255, 255, 255);
	color: rgb(26, 16, 84);
    }</string>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::ButtonSymbols::UpDownArrows</enum>
    </property>
    <property name="minimum">
     <number>2025</number>
    </property>
    <property name="maximum">
     <number>2050</number>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(26, 16, 84);</string>
   </property>
   <property name="text">
    <string>Select Date</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reportFormGenerateBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(37, 95, 241); 
    color: rgb(255, 255, 255);
    border-radius: 10px;
    font: 'Chakra Petch';
}

QPushButton:hover {
    background-color: rgb(58, 115, 255); 
}

QPushButton:pressed {
    background-color: rgb(20, 70, 200);   
}
</string>
   </property>
   <property name="text">
    <string>Generate Report</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
